pkg/testmachinery/metadata/types.go

```diff
@@ -30,6 +30,7 @@ type Metadata struct {
 	OperatingSystemVersion    string `json:"operating_system_version,omitempty"`
 	Zone                      string `json:"zone,omitempty"`
 	AllowPrivilegedContainers *bool  `json:"allow_privileged_containers,omitempty"`
+	ShootAnnotations          string `json:"shoot_annotations,omitempty"`
```

Comment:
I think for the elasticserach serach it would be better if we had the real map available not just the string.
Then searching would be better.

Reply:
You're right, I was too lazy to add the respective unmarshal function to convert back from string to the map :)
Done now.